src/Entity/Geo/GeoTrait.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+namespace App\Entity\Geo;
+
+use Doctrine\ORM\Mapping as ORM;
+
+trait GeoTrait
+{
+    /**
+     * @var int|null
+     *
+     * @ORM\Id
+     * @ORM\Column(type="integer", options={"unsigned": true})
+     * @ORM\GeneratedValue
+     */
+    private $id;
+
+    /**
+     * @var string|null
+     *
+     * @ORM\Column
```

Comment:
what about making this column `unique` here and avoid all `@ORM\UniqueConstraint(name="...", columns="code")`
```
* @ORM\Column(unique=true)
```